extensions/notebook/src/test/book/book.test.ts

```diff
@@ -28,23 +31,24 @@ export interface IExpectedBookItem {
 	nextUri?: string | undefined;
 }
 
-export function equalBookItems(book: BookTreeItem, expectedBook: IExpectedBookItem): void {
-	should(book.title).equal(expectedBook.title);
+export function equalBookItems(book: BookTreeItem, expectedBook: IExpectedBookItem, errorMsg?: string): void {
+	should(book.title).equal(expectedBook.title, 'Book titles do not match, expected ' + expectedBook.title + ' and got ' + book.title);
```

Comment:
Using interned strings is a lot easier to read

`` `Book titles do not match, espected ${expectedBook.title} and got ${book.title}` ``